src/client/src/app/components/events/event-details/event-details.component.ts

```diff
@@ -207,6 +219,23 @@ export class EventDetailsComponent {
     });
   }
 
+  protected updateExternalUri() {
+    const event = this.event();
+    if (!event) return;
+
+    this._store.dispatch(
+      updateEventAction({
+        eventId: event.id,
+        externalUri:
+          this.externalUri() != null ? (this.externalUri() as string | undefined) : undefined,
+      })
+    );
+  }
+
+  protected openExternalUri() {
+    window.location.href = this.externalUri() as string;
```

Comment:
on mobile, when providing a valid Facebook link, it jumps directly to the Facebook app. the currently open page inside the Minigolf App will stay open.
in edge on windows, it will replace the tab with the Facebook page